app/src/main/java/com/codesoom/assignment/controllers/UserController.java

```diff
@@ -0,0 +1,59 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.UserService;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/users")
+@CrossOrigin
+public class UserController {
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    /**
+     *  userData에 user을 생성하고 user가 생성되면 201을 응답한다.
+     *
+     * @param userData 생성된 user 정보
+     * @return 생성된 user
+     */
+
+    @PostMapping
```

Comment:
JavaDoc과 JavaDoc이 설명하는 대상 사이에 공백 라인을 넣지 마세요.
```suggestion
    /**
     *  userData에 user을 생성하고 user가 생성되면 201을 응답한다.
     *
     * @param userData 생성된 user 정보
     * @return 생성된 user
     */
    @PostMapping
```

Reply:
수정하겠습니다